pyproject.toml

```diff
@@ -27,6 +27,7 @@ classifiers = [
   "Topic :: Scientific/Engineering",
 ]
 dependencies = [
+  "numpy <=1.24.3",
```

Comment:
I'd suggest moving this to the CI resolution, if required; we'll patch this upstream in short order, and then make a new release!

Reply:
right, I did this as a temporary sanity check just to see if GH actions would pass :) - gone now!